app/views/projects/_table.html.erb

```diff
@@ -1,25 +1,33 @@
 <table class="table" data-toggle="datatable">
   <thead>
-    <tr>
-      <th><%= User.human_attribute_name('name') %></th>
-      <th><%= User.human_attribute_name('description') %></th>
-      <th><%= User.human_attribute_name('responsible_users') %></th>
-    </tr>
+  <th>Names</th>
+  <th>Description</th>
+  <th>Responsible</th>
+  <th>Actions</th>
   </thead>
   <tbody>
-    <% projects.each do |project| %>
-      <tr>
-        <td><%= link_to project.name, project_path(project) %></td>
-        <td><%= truncate(project.description, length: 100) %></td>
-        <td>
-          <%# https://getbootstrap.com/docs/4.0/content/typography/#inline %>
-          <ul class="list-inline">
-            <% project.responsible_users.each do |user| %>
-              <li class="list-inline-item"><%= user.name %></li>
-            <% end %>
-          </ul>
-        </td>
-      </tr>
-    <% end %> <%# projects.each %>
+  <% projects.each do |project| %>
+    <tr>
+      <td><%= link_to project.name, project_path(project) %></td>
+      <td><%= project.description %></td>
+      <td>
+        <ul class="list-inline mb-0">
+          <% project.responsible_users.each do |user| %>
+            <li class="list-inline-item"><%= link_to user.name, user %></li>
+          <% end %>
+        </ul>
+      </td>
+      <td>
+        <% if project.responsible_users.include? current_user %>
+          <%= link_to fa_icon('trash'),
```

Comment:
The edit link is on the Project#show page. I'll move the delete button there too.